server/src/routes/api/users.js

```diff
@@ -0,0 +1,26 @@
+import express from "express";
+import models from "../../models";
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+	const users = await models.User.findAll();
+	return res.json({ users });
+});
+
+router.post("/", async (req, res) => {
+	let user = await models.User.Create(req.body);
+	return res.json({ user });
+});
+
+router.get("/:id", async (req, res) => {
+	let user = await models.User.findByPk(req.param.id);
+	return res.json({ user });
+});
+
+router.put("/:id", async (req, res) => {
+	let newUser = await models.User.update(req.body, { where: req.params.id });
+	return res.json({ user: newUser });
+});
```

Comment:
The actual specific user will probably be provided by auth0 identity rather than the param id, so we can just return User model with a dummy ID for now and get rid of the route param, and then update that id to be provided by auth0 once that gets fleshed out.